website/docs/getting-started/setup/instance-configuration/authentication/google-login.md

```diff
@@ -7,64 +7,63 @@ sidebar_position: 1
+  ![Click to expand](</img/Google_Oauth_Creds_2.png>)
 
-**5. Update the**[ **instance configuration**](../) **with the following fields**:
+## Configure your instance  
 
-  * **Configure via UI (recommended)**:
+Update the [**instance configuration**](../) with the following fields:
 
-  On your self-hosted instance, navigate to the [Admin Settings page](/getting-started/setup/instance-configuration/). Under the **Authentication** tab, you can find fields for providing your Client ID and Client Secret generated in the previous steps.
+1. **Configure via UI (recommended):**  On your self-hosted instance, navigate to the [Admin Settings page](/getting-started/setup/instance-configuration/). Under the **Authentication** tab, you can find fields for providing your Client ID and Client Secret generated in the previous steps.
 
-  ![](/img/as_google_auth_config.png)
+![](/img/as_google_auth_config.png)
 
-  Once you've entered your provided the Client ID and Client Secret, click the **Save & Restart** button at the bottom of the page. This prompts your Appsmith instance with the new configuration in effect.
+Once you've entered your Client ID and Client Secret, click the **Save & Restart** button at the bottom of the page. This prompts your Appsmith instance with the new configuration in effect.
 
-  * **Configure via docker.env file**:
-    
-    On the machine hosting the Appsmith instance, find your `docker.env` file located in `<project-root>/stacks/configuration/`. Update the values for the following keys:
+2. **Configure via docker.env file**: On the machine hosting the Appsmith instance, find your `docker.env` file located in `<project-root>/stacks/configuration/`. Update the values for the following keys:
 
-    ```bash
-    # Example configuration in docker.env file
-    # ******** Google OAuth ********
-    APPSMITH_OAUTH2_GOOGLE_CLIENT_ID=YOUR_CLIENT_ID
-    APPSMITH_OAUTH2_GOOGLE_CLIENT_SECRET=YOUR_CLIENT_SECRET
-    APPSMITH_SIGNUP_ALLOWED_DOMAINS=exampledomain.com
-    # ******************************
-    ```
+  ```bash
+  # Example configuration in docker.env file
+  # ******** Google OAuth ********
+  APPSMITH_OAUTH2_GOOGLE_CLIENT_ID=YOUR_CLIENT_ID
+  APPSMITH_OAUTH2_GOOGLE_CLIENT_SECRET=YOUR_CLIENT_SECRET
+  APPSMITH_SIGNUP_ALLOWED_DOMAINS=exampledomain.com
+  # ******************************
+   ```
 
     * Configure the `APPSMITH_OAUTH2_GOOGLE_CLIENT_ID` and `APPSMITH_OAUTH2_GOOGLE_CLIENT_SECRET` fields with the client ID and client secret generated in the step above
     * Configure `APPSMITH_SIGNUP_ALLOWED_DOMAINS` with a second-level domain name **`(ex: abc.com)`**, to only allow users with an email address from that domain name to log in **`(ex: john@abc.com)`**
-      * **Note**: you can provide several domain names using a comma-separated list
+    * **Note**: you can provide several domain names using a comma-separated list
     * [Restart the Appsmith instance](../)
 
 ---
 
 :::caution
```

Comment:
this callout should be on the top right after the heading 'Configure Appsmith instance'